### DISCUSSION TITLE
Is there a limit on private repositories or external contributors for GitHub organizations that use the free plan?

### DISCUSSION THREAD

**Discussion by gr2m** [community/community#60002] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey friends 👋🏼 

I'm helping friends with a GitHub App that they want to build, which would potentially create thousands of private repositories and invite a different user to each of them. This is for a non-profit project, the plan is to use the free plan for an organization.

I'm quite sure there is no limit on private repositories. But is there a limit on external contributors?

Cheers!

**Comment by nirav-gajera** [comment, marked as answer]
There is no specific limit on the number of external contributors who can contribute to a private repository in a free organization. Contributors are users who can create pull requests and contribute to the repository without having read or write access.

If the organization needs to have more than 3 external collaborators with read/write/admin access to private repositories, they would need to consider upgrading to a paid plan that supports more collaborators.

Collaborators: Free organizations are limited to 3 external collaborators for private repositories.
